tests/001-core/test-password.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+class Current_Password_Change extends WP_UnitTestCase {
```

Comment:
Minor - it would be good to name this class so that it's obvious it's a test case.
```suggestion
class Current_Password_Change_Test extends WP_UnitTestCase {
```